Beams/MRML/vtkMRMLRTBeamNode.cxx

```diff
@@ -57,6 +58,155 @@ static const char* MLCPOSITION_REFERENCE_ROLE = "MLCPositionRef";
 static const char* DRR_REFERENCE_ROLE = "DRRRef";
 static const char* CONTOUR_BEV_REFERENCE_ROLE = "contourBEVRef";
 
+//------------------------------------------------------------------------------
+namespace

```

Comment:
Why do you use a namespace here? It would be much better making this part of the class

Reply:
By means of vtkInternal, or just private?

Reply:
If there is no internal class yet, then no need to bother, just private. Otherwise internal please.